l10n_br_sale/models/sale.py

```diff
@@ -51,25 +51,25 @@ def _amount_line_tax(self, line):
         price = line._calc_line_base_price()
         qty = line._calc_line_quantity()
         for computed in line.tax_id.compute_all(
-                price, qty, product=line.product_id,
-                partner=line.order_id.partner_invoice_id,
-                fiscal_position=line.fiscal_position)['taxes']:
+                price, quantity=qty, product=line.product_id,
+                partner=line.order_id.partner_invoice_id)['taxes']:
             tax = self.env['account.tax'].browse(computed['id'])
-            if not tax.tax_code_id.tax_discount:
+            if not tax.tax_group_id.tax_discount:
                 value += computed.get('amount', 0.0)
         return value
 
-    def _invoiced_rate(self, cursor, user, ids, name, arg, context=None):
+    @api.multi
+    def _invoiced_rate(self, name, arg):
```

Comment:
the field invoiced_rate has been dropped since Odoo 9.0. So invoiced_rate and this method should be removed from the localization; it's now useless.